src/main/antlr4/Manifold.g4

```diff
@@ -89,14 +92,23 @@ lvalue:
   ;
 
 // TODO: declarations as expressions
-expression: /* declaration | */ rvalue;
+expression:
+  rvalue
+  // | declaration
+  ;
+
+statement:
+    expression #ExpressionStatment
+    | 'import' STRING_VALUE #ImportStatement
```

Comment:
I supposed, but I can't think of any language that does it like that. Every other language has something like `import '../digital' as HalfAdder'` or `from '../digital' import HalfAdder'`